db/seeds.rb

```diff
@@ -5,3 +5,42 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+require 'faker'
+
+Booking.destroy_all
+Band.destroy_all
+User.destroy_all
+
+
+User.create!(first_name: Faker::Name.first_name, last_name: Faker::Name.last_name, email: Faker::Internet.email, password: Faker::Internet.password)
+User.create!(first_name: Faker::Name.first_name, last_name: Faker::Name.last_name, email: Faker::Internet.email, password: Faker::Internet.password)
+User.create!(first_name: Faker::Name.first_name, last_name: Faker::Name.last_name, email: Faker::Internet.email, password: Faker::Internet.password)
+
+Band.create!(name: "Pink Floyd", 
+  description: "An English rock band that became famous in the 1960s and 1970s. They performed for more than 30 years, and have sold more than 200 million albums.", 
+  genre: "Rock", 
+  photo_url:"https://upload.wikimedia.org/wikipedia/id/thumb/f/f3/PinkFloyd-album-piperatthegatesofdawn.jpg/640px-PinkFloyd-album-piperatthegatesofdawn.jpg",
+  user: User.first
+)
+Band.create!(name: "Chicago", 
+    description: "One of the famous American jazz rock bands formed in Chicago, Illinois, in 1967.", 
+    genre: "Jazz", 
+    photo_url:"https://upload.wikimedia.org/wikipedia/commons/1/16/ChicagoCollage2-1000.jpg",
+    user: User.first
```

Comment:
Currently all the bands belong to 1 user (User.first), which isn't the most helpful in terms of testing. Can you assign this band to the second user you create?